VndbSharp/Enums/VndbFlags.cs

```diff
@@ -21,13 +21,15 @@ public enum VndbFlags
 		Stats = 1 << 5,
 		[Description("screens")]
 		Screens = 1 << 6,
-		[Description("vns")]
-		VisualNovels = 1 << 7,
+		[Description("vn")]
+		VisualNovel = 1 << 7,
```

Comment:
Not really sure why you need to change the flag for this, when adding a new one anyways?